src/js/constants.js

```diff
@@ -1,5 +1,6 @@
 // Will get replaced by gulpfile with relevant environment url
 export const STUY_SPEC_API_URL = "__API_URL_HERE__";
+//export const STUY_SPEC_API_URL = "https://api.stuyspec.com/";
```

Comment:
Just delete this line, since we use the `DEV_API` and `PROD_API` variables in the dotenv.